sites/partners/src/components/listings/PaperListingForm/index.tsx

```diff
@@ -166,7 +168,21 @@ const ListingForm = ({ listing, editMode, setListingName }: ListingFormProps) =>
   }, [listing?.units, listing?.listingEvents, setUnits, setOpenHouseEvents])
 
   // eslint-disable-next-line @typescript-eslint/unbound-method
-  const { getValues, setError, clearErrors, reset } = formMethods
+  const { getValues, setError, clearErrors, reset, watch } = formMethods
+
+  const selectedJurisdiction = watch("jurisdictions.id")
+
+  useEffect(() => {
+    const newFeatureFlags = profile.jurisdictions?.reduce((featureFlags, juris) => {
```

Comment:
Just a nit, wondering if this makes sense as a helper that lives somewhere else

Reply:
As we get more use cases I will refactor this to know the best place to put it